bridges/MastodonBridge.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+class MastodonBridge extends FeedExpander {
+
+	const MAINTAINER = 'husim0';
+	const NAME = 'Mastodon Bridge';
+	const CACHE_TIMEOUT = 900; // 15mn
+	const DESCRIPTION = 'Returns toots';
+	const URI = 'https://mastodon.social';
+
+	const PARAMETERS = [[
```

Comment:
Please use the `array()` notation instead of `[]` to stay consistent with other bridges.